enterprise/api_client/discovery.py

```diff
@@ -72,6 +79,36 @@ def __init__(self, user):
         self.user = user
         self.client = course_discovery_api_client(user)
 
+    def get_search_results(self, querystring=None):
+        """
+        Return a search result from all data.
+
+        Returns:
+            dict: API response with search results
+
+        """
+        return self._load_data(
+            self.SEARCH_ALL_ENDPOINT,
+            default=[],
+            querystring=querystring,
+        )
+
+    def get_paginated_search_results(self, querystring=None):
+        """
+        Return paginated search result from all data.
+
+        Returns:
+            dict: API response with search results, as well as links to next and previous pages.
+
+        """
+        return self._load_data(
+            self.SEARCH_ALL_ENDPOINT,
+            default=[],
+            querystring=querystring,
+            traverse_pagination=False,
```

Comment:
@UmanShahzad What does `traverse_pagination` mean? (You didn't introduce that parameter so no need to change anything; I'm just asking because its name is a little confusing to me.)